bentoml/server/api_server.py

```diff
@@ -204,6 +210,7 @@ def setup_routes(self):
         /healthz        Health check ping
         /feedback       Submitting feedback
         /metrics        Prometheus metrics endpoint
+        /metadata       BentoService Artifact Metadata
```

Comment:
Also update the `get_open_api_spec_json` method in the `open_api.py` file to include this new endpoint?